### DISCUSSION TITLE
Questions on 'model_devi_jobs'

### DISCUSSION THREAD

**Discussion by yunjian-hash** [deepmodeling/dpgen#1050] [Q&A]
Dear developers,

I want to use four different sys_configs to run four different MD trajectories in the first iteration and collect 200 fp for each of the trajectories (that is 800 fp in one iteration), is it appropriate to use the same "_idx": "00",or use  from "_idx": "00" to "_idx": "03" with "skip_train":true?  Part of my param.json file is shown below. Thanks!

 "model_devi_jobs": [
	{   "sys_idx": [0],"trj_freq": 10,"_idx": "00",
        "template":{ "lmp": "lmp/Si_dissolution/input.lammps", "plm": "lmp/Si_dissolution/input.plumed" },
        "rev_mat":{ "lmp": {"V_NSTEPS": [30000], "V_TEMP": [300], "V_PRES": [0]}, "plm": {"V_TEMP":  [300], "V_STRIDE": [50]} }},
	{   "sys_idx": [1],"trj_freq": 10,"_idx": "00",
        "template":{ "lmp": "lmp/Si_dissolution_removeCa_firstlayer/input.lammps", "plm": "lmp/Si_dissolution_removeCa_firstlayer/input.plumed" },
        "rev_mat":{ "lmp": {"V_NSTEPS": [30000], "V_TEMP": [300], "V_PRES": [0]}, "plm": {"V_TEMP":  [300], "V_STRIDE": [50]} }},
	{   "sys_idx": [2],"trj_freq": 10,"_idx": "00",
        "template":{ "lmp": "lmp/Si_dissolution_sec/input.lammps", "plm": "lmp/Si_dissolution_sec/input.plumed" },
        "rev_mat":{ "lmp": {"V_NSTEPS": [30000], "V_TEMP": [300], "V_PRES": [0]}, "plm": {"V_TEMP":  [300], "V_STRIDE": [50]} }},
	{   "sys_idx": [3],"trj_freq": 10,"_idx": "00",
        "template":{ "lmp": "lmp/Si_Ca_add/input.lammps", "plm": "lmp/Si_Ca_add/input.plumed" },
        "rev_mat":{ "lmp": {"V_NSTEPS": [30000], "V_TEMP": [300], "V_PRES": [0]}, "plm": {"V_TEMP":  [300], "V_STRIDE": [50]} }}
    ],
    "fp_style": "cp2k",
    "shuffle_poscar": false,
    "fp_task_max": 200,
    "fp_task_min": 30,
    "fp_pp_path": ".",
    "fp_pp_files": [],
    "external_input_path": "/data/C3S_dissolution/Ca/cp2k/cp2k.inp"
  }

**Comment by njzjz** [comment]
`_idx` is a comment and does not have any actual effect.
